src/ui/optionsdialog.cc

```diff
@@ -79,20 +82,17 @@ void OptionsDialog::accept() {
       color_3d_begin_button_->getColor());
```

Comment:
I know this is not your code, but shouldn't these be slots of HexEdit & Visualization connected to settingsChanged signal?

Reply:
No, as explained in private.